src/Api/Models/Response/ConfigResponseModel.cs

```diff
@@ -61,6 +67,43 @@ public class EnvironmentConfigResponseModel
     public string Sso { get; set; }
 }
 
+public class PushSettings
+{
+    private readonly bool _webPushEnabled;
+    private readonly string _vapidPublicKey;
+    public PushTechnologyType PushTechnology
+    {
+        get
+        {
+            if (VapidPublicKey != null)
+            {
+                return PushTechnologyType.WebPush;
+            }
+            return PushTechnologyType.SignalR;
+        }
+    }
+    /// <summary>
+    /// Only for use when PushTechnology is WebPush.
+    /// </summary>
+    public string VapidPublicKey
+    {
+        get
+        {
+            if (_webPushEnabled)
+            {
+                return _vapidPublicKey;
+            }
+            return null;
+        }
+    }
+
+    public PushSettings(IFeatureService launchDarklyFeatureService, IGlobalSettings globalSettings)
+    {
+        _webPushEnabled = launchDarklyFeatureService.IsEnabled(FeatureFlagKeys.WebPush);
```

Comment:
:art: While there's a lot of local optimizations and caching down at the client here via the LD SDK, you're technically causing another query by asking for this when you already have all the values with the earlier `GetAll`. Since this API method is used a ton, it would be good to just use that collection of values and not ask again.

Reply:
See 32f64.

I actually don't really like this change as it forces a type conversion for the `object`-valued flag dictionary